smallcloudai/refact-chat-js
# Group sequential diffs.

# WIP: Group sequential diffs and a apply all button.

## Description

Adds an apply all to multiple diff messages.

## TODO:
currently the lsp seems to break when asking this prompt so I'll need to wait to properly run it :/


## Type of change




## How to Test

1. Prompt with `rename frog class to bird class with out renaming the files`
2. It should respond with some diffs,
3. Apply and unapply these diffs using the `Apply All` and `Unapply All` buttons.

## Screenshots (if applicable)




## Checklist




## Linked Issues

https://github.com/orgs/smallcloudai/projects/5/views/1?pane=issue&itemId=79900913

## Additional Notes



I had to re-sync this branch after merging the `apply warning` messages branch in to alpha
